checker/jdk/nullness/src/java/util/concurrent/BlockingDeque.java

```diff
@@ -411,7 +411,7 @@ boolean offerLast(E e, long timeout, TimeUnit unit)
      * @throws NullPointerException if the specified element is null
      *         (<a href="../Collection.html#optional-restrictions">optional</a>)
      */
-    boolean removeFirstOccurrence(Object o);
+    boolean removeFirstOccurrence(@Nullable Object o);
```

Comment:
I don't think we want `@Nullable` on these 4 methods in `BlockingDeque`, nor on the 2 in `BlockingQueue`.